backend/shared/shared/audit_log/tasks.py

```diff
@@ -0,0 +1,45 @@
+import logging
+from datetime import timedelta
+
+from django.conf import settings
+from django.utils import timezone
+from elasticsearch import Elasticsearch
+
+from shared.audit_log.models import AuditLogEntry
+
+ES_STATUS_CREATED = "created"
+LOGGER = logging.getLogger(__name__)
+
+
+def send_audit_log_to_elastic_search():
+    if not (
+        settings.ELASTICSEARCH_CLOUD_ID
+        and settings.ELASTICSEARCH_API_ID
+        and settings.ELASTICSEARCH_API_KEY
+    ):
+        LOGGER.warning(
+            "Trying to send audit log to Elasticsearch without proper configuration, process skipped"
+        )
+        return
+    es = Elasticsearch(
+        cloud_id=settings.ELASTICSEARCH_CLOUD_ID,
+        api_key=(settings.ELASTICSEARCH_API_ID, settings.ELASTICSEARCH_API_KEY),
+    )
+    entries = AuditLogEntry.objects.filter(is_sent=False)
+
+    for entry in entries:
+        rs = es.index(
+            index=settings.ELASTICSEARCH_APP_AUDIT_LOG_INDEX, body=entry.message
```

Comment:
Should we also index the django ID here so it is easier to find the correct logs from ES if that is ever needed.

```
rs = es.index(
            index=settings.ELASTICSEARCH_APP_AUDIT_LOG_INDEX, id=entry.id, body=entry.message
)
```

Reply:
That'd work too. By default, an index id will be generated but having the index id as a reference to the object id might be useful somehow.

Reply:
Yea, I was thinking if there is some special case in which it would be useful. For example there were some unnecessary logs sent that need to be deleted.